### PR TITLE
docs: add warning for azure blobs with credentials

### PR DESCRIPTION
**Pull request by Jonas1312**

### CODE REVIEW

**Review by PierreLeveau — APPROVED** [review]

File: src/kili/entrypoints/mutations/data_connection/__init__.py [hunk 1]
```diff
@@ -99,6 +100,11 @@ def synchronize_cloud_storage_connection(
 
         Returns:
             A dict with the DataConnection ID.
+
+        !!! warning "Azure Blob Storage"
+            This method currently does not work for Azure cloud storage connection using credentials mode.
+            Use service account mode instead or use the Kili app to synchronize the data connection.
```
Comment (theodu) [inline src/kili/entrypoints/mutations/data_connection/__init__.py:106]: what is the service account mode ?

Comment (Jonas1312) [reply src/kili/entrypoints/mutations/data_connection/__init__.py:106]: https://docs.kili-technology.com/docs/creating-an-azure-blob-storage-integration#add-service-account

Comment (PierreLeveau) [reply src/kili/entrypoints/mutations/data_connection/__init__.py:106]: Maybe we can add this link in the docstring?

Comment (PierreLeveau) [reply src/kili/entrypoints/mutations/data_connection/__init__.py:106]: ah ah missed that

**Pull request merged**